docs/src/external_mctp_vdm_cmds.md

```diff
@@ -0,0 +1,508 @@
+| Vendor-Defined Message Body | 3:M     | Vendor-defined message body, 0 to M bytes.                              |
+
+The Vendor ID is a 16-bit unsigned integer, described in the PCI 2.3 specification. The value identifies the device manufacturer. The message body and content are described in the sections below.
+
+### Message Format
+
+This section describes the MCTP message format used to support Caliptra subsystem external command protocol. The request/response message body encapsulates the Vendor Defined MCTP message within the MCTP transport. Details of MCTP message encapsulation can be found in the MCTP Base Specification. The MCTP Get Vendor Defined Message Support command allows discovery of the vendor-defined messages supported by an endpoint. This discovery process identifies the vendor organization and the supported message types. The format of this request is specified in the MCTP Base Specification.
+
+For the Caliptra external command protocol, the following information is returned in response to the MCTP Get Vendor Defined Message Support request:
+- **Vendor ID Format**: `0`
+- **PCI Vendor ID**: `0x1414`
+- **Command Set Version**: `4`
+
+The following table provides detailed descriptions of the fields used in the Caliptra external command protocol:
+
+| Field Name           | Description                                                                                                   |
+|----------------------|---------------------------------------------------------------------------------------------------------------|
+| **IC**               | (MCTP Integrity Check bit) Indicates whether the MCTP message is covered by an overall MCTP message payload integrity check. |
+| **Message Type**     | Indicates an MCTP Vendor Defined Message.                                                                     |
+| **MCTP PCI Vendor**  | ID for PCI Vendor. Caliptra messages use the Microsoft PCI ID of `0x1414`.                                    |
+| **Request Type**     | Distinguishes between request and response messages: set to `1` for requests, and `0` for responses. |
+| **Crypt**            | Indicates whether the Message Payload and Command are encrypted.                                              |
+| **Command Code**     | The command ID for the operation to execute.                                                                  |
+| **Msg Integrity Check** | Represents the optional presence of a message type-specific integrity check over the contents of the message body. If present (indicated by the IC bit), the Message Integrity Check field is carried in the last bytes of the message body. |
+
+The following table describes the MCTP message format used in the Caliptra external command protocol:
+
+*Table: MCTP Vendor Defined Message Format*
+<img src="images/mctp_vdm_format.svg" alt="Vendor defined message format" align="center" />
```

Comment:
Where did this image come from? Is it original, or do we need to cite it? (We don't want to add things that are copyrighted by someone else.)